variables/_mixins.scss

```diff
@@ -0,0 +1,10 @@
+@mixin flexCenter {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+@mixin imageSpecs {
+  width: 35rem;
+  height: 35rem;
+}

```

Comment:
Instead of doing a mixin for these kinds of things, do a template. Templates are like a class but can be reused wherever you want.

Templates looks like this:

```
%imageSpecs {
   width: 35rem;
   height: 35rem;
 }
 ```

and then you use it in the class you want like so:

```
.container {
  @extend %imageSpecs;
  ...
}
```

But then, there's the question: Is this code going to be reused somewhere else? because if not, it's not necessary to do any of this